.github/workflows/CI.yml

```diff
@@ -55,12 +50,19 @@ jobs:
       - name: Run playwright example-next
         run: |
           set -e
+          echo "NEXT_PUBLIC_LOCALHOST_URL=http://localhost:8545" > .env.local
+          echo "NEXT_PUBLIC_MAINNET_URL=${MAINNET_URL}" >> .env.local
           pnpm dev &
           xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- pnpm test:playwright
         env:
           MAINNET_URL: ${{ secrets.MAINNET_PROVIDER_URL }}
           DEBUG: usedapp*
         working-directory: packages/example-next
+      - name: Cleanup
```

Comment:
We use github-hosted runners so no need for cleanup, but we can leave it in case we switch to self-hosted.